gcp-observability/src/test/java/io/grpc/gcp/observability/logging/GcpLogSinkTest.java

```diff
@@ -64,18 +63,18 @@ public class GcpLogSinkTest {
       "pod_name", "app1-6c7c58f897-n92c5");
   private static final Map<String, String> customTags = ImmutableMap.of("KEY1", "Value1",
       "KEY2", "VALUE2");
-  private static final long flushLimit = 10L;
-  // gRPC is expected to alway use this log name when reporting to GCP cloud logging.
+  private static final long FLUSH_LIMIT = 10L;
+  // gRPC is expected to always use this log name when reporting to GCP cloud logging.
   private static final String expectedLogName =
       "microservices.googleapis.com%2Fobservability%2Fgrpc";
-  private final long seqId = 1;
-  private final String destProjectName = "PROJECT";
-  private final String serviceName = "service";
-  private final String methodName = "method";
-  private final String authority = "authority";
-  private final Duration timeout = Durations.fromMillis(1234);
-  private final String rpcId = "d155e885-9587-4e77-81f7-3aa5a443d47f";
-  private final GrpcLogRecord logProto = GrpcLogRecord.newBuilder()
+  private static final long seqId = 1;
+  private static final String destProjectName = "PROJECT";
+  private static final String serviceName = "service";
+  private static final String methodName = "method";
+  private static final String authority = "authority";
+  private static final Duration timeout = Durations.fromMillis(1234);
+  private static final String rpcId = "d155e885-9587-4e77-81f7-3aa5a443d47f";
+  private static final GrpcLogRecord LOG_PROTO = GrpcLogRecord.newBuilder()
```

Comment:
All of the above are constants so why not follow the coding convention and declare in all upper-case? From `locationTags` to `rpcId`

Reply:
Updated.